### ISSUE TITLE
Gatling 2.3 fail to terminate system in 2 seconds

### ISSUE THREAD

**Issue by huaxiawang** [gatling/gatling#3408]
This issue is occasionally happening.
We are using Gradle gatling plugin to testing performance while building it, although the simulation finished successfully, it timed out after that with following exception.

```
---- PatientMeasureReportAdded--------------------------------------------
[##########################################################################]100%
          waiting: 0      / active: 0      / done:4     
================================================================================

Simulation PatientMeasureReportAddedSimulation completed in 79 seconds
Exception in thread "main" java.util.concurrent.TimeoutException: Futures timed out after [2 seconds]
	at scala.concurrent.impl.Promise$DefaultPromise.ready(Promise.scala:255)
	at scala.concurrent.impl.Promise$DefaultPromise.result(Promise.scala:259)
	at scala.concurrent.Await$.$anonfun$result$1(package.scala:215)
	at scala.concurrent.BlockContext$DefaultBlockContext$.blockOn(BlockContext.scala:53)
	at scala.concurrent.Await$.result(package.scala:142)
	at io.gatling.app.Gatling$.start(Gatling.scala:62)
	at io.gatling.app.Gatling$.fromArgs(Gatling.scala:43)
	at io.gatling.app.Gatling$.main(Gatling.scala:35)
	at io.gatling.app.Gatling.main(Gatling.scala)
:gatlingRun-PatientMeasureReportAddedSimulation FAILED 
```

Since the 2 seconds is hard coded in `Gatling.scala`, I am not able to increase it. Can you make it configurable?

**Comment by slandelle** [comment]
> Can you make it configurable?

Actually, why don't YOU make it configurable and contribute with a PR?

FYI, it's been raised to 5 seconds in master.

**Comment by saint1991** [comment]
What key is proper for this configuration?

**Comment by schuch** [comment]
Will this be fixed for 2.3.x as well?

**Comment by moozo89** [comment]
Hi, is there a fix for gatling 2.3.x or some workaround?

**Comment by slandelle** [comment]
No. Gatling 2 reached end of life 2 years ago now. You have to upgrade, see https://gatling.io/docs/current/migration_guides/!
